orchestration/dags/data_gcp_dbt/profiles.yml

```
@@ -72,3 +72,37 @@ elementary:
       project: passculture-data-prod
       location: europe-west1
       threads: 2
+
+CI:
+  target: dev
+  outputs:
+    dev:
+      dataset: CI_dev
```

```suggestion
      dataset: ci_dev
```
en minuscule pour etre cohérent avec notre env BQ ?